CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.2.0
+* Validate Puppet version_requirement [#99](https://github.com/voxpupuli/metadata-json-lint/issues/99)
+* Add optional check to validate the Puppet Agent Version is not EOL [#100](https://github.com/voxpupuli/metadata-json-lint/pull/100)
```

Comment:
I'm not sure I'd understand this if I didn't review the PR. How about:
Add `--strict-puppet-version` to do strict Puppet version requirement checks. It doesn't allow open ended or mixed requirements. It also checks if it allows EOL Puppet versions. Currently that means it'll complain if the lower bound allows versions older than 4.10.0.

Reply:
Good point. I updated the changelog